### PR TITLE
updated to remove duplicates

### PR DESCRIPTION
**Pull request by kiyoshi626**
struggled a little bit when i got to remove duplicates

### CODE REVIEW

File: callBackPractice/callBackPractice.js [hunk 1]
```diff
@@ -20,45 +20,81 @@
...
+};
+
 contains(foods, 'ribeye', (result) => {
   console.log(result ? 'ribeye is in the array' : 'ribeye is not in the array');
 });
 
 // Write a function called removeDuplicates that removes all duplicate values from the given array.
 // Pass the array to the callback function.  Do not mutate the original array.
 
+// const foods2 = [food[0]]
```
Comment (ryan-hamblin) [inline callBackPractice/callBackPractice.js]: If you comment out this code, does it still work? It seems like you're using this on line 94, you might need it.

**Review by ryan-hamblin — COMMENTED** [review]
Great job. Just had one thing to say!

**Comment by kiyoshi626** [comment]
Updated with reverseCase

**Comment by kiyoshi626** [comment]
Updated with latest code challenge

**Pull request closed without merging**